dags/ecocounter_open_data.py

```diff
@@ -106,46 +96,93 @@ def reminder_message(ds = None, **context):
     wait_till_10th.doc_md = """
     Wait until the 10th day of the month to export data. Alternatively mark task as success to proceed immediately.
     """
+    
+    @task()
+    def get_years(ds=None):
```

Comment:
did we add this for mapped task naming? It was using ds before right?

Reply:
Yes, the `insert_and_download_data` task_group is mapped over the output of `get_years`. The purpose is to run the exports for last two months (which may be two separate years), since new data frequently arrives within ~30 days. 
![image](https://github.com/user-attachments/assets/cff61004-275d-4ec8-be6b-841a597a1ce6)